expression-tracker/server.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'open3'
+require 'mqtt'
+require 'json'
+require_relative 'helper'
+include Helper
+
```

Comment:
This `mqtt` flag, the `mqtt_*` settings and the `--mqtt` option on line 14 are perfect candidates for the use of `OptionParser`. 

```ruby
require 'optparse'

@options = {
  mqtt: false,
  mqtt_broker: ENV["MQTT_BROKER"] || "mqtt://localhost",
  mqtt_topic: ENV["MQTT_TOPIC"]] || "PIT"
}
OptionParser.new do |opts|
  opts.banner = "Usage: server.rb [options]"

 opts.on("-m", "--mqtt", Boolean, "Use MQTT") do |i|
    @options[:mqtt] = true
 end
# etc...
```